v21.1/backup.md

```diff
@@ -147,78 +142,74 @@ Per our guidance in the [Performance](#performance) section, we recommend starti
 
 ### Backup a cluster
 
- To backup a full cluster:
+To take a full backup a cluster:
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> BACKUP TO \
-'gs://acme-co-backup/test-cluster' \
```

Comment:
~This feeds into my curiosity of whether BACKUP .. TO is gone or merely deprecated (no action needed, just musing)